README.md

```diff
@@ -122,6 +122,27 @@ Only the client devices have information about users, their contacts and groups.
 
 See [SimpleX whitepaper](https://github.com/simplex-chat/simplexmq/blob/master/protocol/overview-tjr.md) for more information on platform objectives and technical design.
 
+## Privacy: technical details and limitations
+
+SimpleX Chat is a work in progress, we are releasing improvements as they are ready. You have to decide if the current state is good enough for your scenarios.
+
+What is already implemented:
+
+1. Instead of user profile identifiers used by all other platforms, even the most private, SimpleX uses pairwise per-queue identifiers (2 addresses for each message queue, 2 queues in each connection between the users) . It makes understanding of network graph more difficult on the application level, as for `n` users you have `n * (n-1)` message queues.
+2. End-to-end encryption in each message queue using [NaCl cryptobox](https://nacl.cr.yp.to/box.html). This is added to allow redundancy in the future (passing each message via several servers), to avoid having the same ciphertext in different queues (that would only be visible to the attacker if TLS is compromised). The encryption keys used for this level are not rotated, instead we are planning to replace the queues. Curve25519 keys are used for key negotiation.
```

Comment:
"rotate" is better? "replace" sounds like one-time
```suggestion
2. End-to-end encryption in each message queue using [NaCl cryptobox](https://nacl.cr.yp.to/box.html). This is added to allow redundancy in the future (passing each message via several servers), to avoid having the same ciphertext in different queues (that would only be visible to the attacker if TLS is compromised). The encryption keys used for this level are not rotated, instead we are planning to rotate the queues. Curve25519 keys are used for key negotiation.
```